src/number-formatting/format.js

```diff
@@ -12,6 +12,37 @@ function formatNegativeNumber(str, options) {
 }
 
 /**
+ * converts a number to a decimal string if it is on scientific notation
+ * (method taken from https://gist.github.com/jiggzson/b5f489af9ad931e3d186)
```

Comment:
lets not reference this gist since we are making changes to make it more readable (not sure what a link adds?)